homeassistant/components/renson/number.py

```diff
@@ -0,0 +1,78 @@
+"""Platform to control a Renson ventilation unit."""
+from __future__ import annotations
+
+import logging
+
+from renson_endura_delta.field_enum import FILTER_PRESET_FIELD, DataType
+from renson_endura_delta.renson import RensonVentilation
+
+from homeassistant.components.number import NumberEntity, NumberEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import RensonCoordinator
+from .const import DOMAIN
+from .entity import RensonEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+RENSON_NUMBER_DESCRIPTION = NumberEntityDescription(
+    key="filter_change",
+    translation_key="filter_change",
+    icon="mdi:filter",
+    native_step=1,
+    native_min_value=0,
+    native_max_value=360,
+    entity_category=EntityCategory.CONFIG,
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Renson number platform."""
+
+    api: RensonVentilation = hass.data[DOMAIN][config_entry.entry_id].api
+    coordinator: RensonCoordinator = hass.data[DOMAIN][
+        config_entry.entry_id
+    ].coordinator
+
+    async_add_entities([RensonNumber(RENSON_NUMBER_DESCRIPTION, api, coordinator)])
+
+
+class RensonNumber(RensonEntity, NumberEntity):
+    """Representation of the Renson number platform."""
+
+    _attr_has_entity_name = True
```

Comment:
Just a side note, can this be set at entity level? I don't really see why binary sensor does support it but sensor doesnt.